internal/backend/db/dbgorm/dbgorm.go

```diff
@@ -68,7 +68,11 @@ func New(z *zap.Logger, cfg *Config) (db.DB, error) {
 func (db *gormdb) GormDB() *gorm.DB { return db.db }
 
 func (db *gormdb) Connect(ctx context.Context) error {
-	client, err := gormkitteh.OpenZ(db.z.Named("gorm"), db.cfg, func(cfg *gorm.Config) {
+	var l *zap.Logger
```

Comment:
have you seen https://github.com/autokitteh/autokitteh/blob/460a87efe740a55098028240723f0ce982af8591/internal/backend/gormkitteh/z.go#L27?

Reply:
ENG-1590 looks like a more comprehensive approach to logging, but IIUC it'll require extensive changes throughout AK - that's why it's a TODO.